src/soundbase.cpp

```diff
@@ -166,30 +166,38 @@ QString CSoundBase::SetDev ( const QString strDevName )
         if ( !vsErrorList.isEmpty() )
         {
             // create error message with all details
-            QString sErrorMessage = "<b>" + QString ( tr ( "No usable %1 audio device found." ) ).arg ( strSystemDriverTechniqueName ) + "</b><br>";
+            QString sErrorMessage = "<b>";
```

Comment:
Just declare it here - the floating `"<b>"` makes the code hard to follow.

Reply:
Ok. Still struggeling with the syntax though.